ChiModules/KEigenvalueSolver/IterativeOperations/set_k_source.cc

```diff
@@ -80,36 +74,36 @@ void KEigenvalue::Solver::SetKSource(LBSGroupset& groupset,
       {
         unsigned int ell = m_to_ell_em_map[m].ell;
 
-        int64_t ir = full_cell_view.MapDOF(i,m,0);
-        q_mom      = &q_moments_local[ir];
+        int64_t ir        = full_cell_view.MapDOF(i,m,0);
+        double* q_mom     = &q_moments_local[ir];
 
-        phi_oldp  = &phi_old_local[ir];
-        phi_prevp = &phi_prev_local[ir];
+        double* phi_oldp  = &phi_old_local[ir];
+        double* phi_prevp = &phi_prev_local[ir];
 
         // ----- Loop over groupset groups
         for (int g=gs_i; g<=gs_f; g++)
         {
           // ----- Contribute scattering
-          inscat_g = 0.0;
-          if ((ell < xs->transfer_matrix.size()) && (!suppress_phi_old) )
+          double inscat_g = 0.0;
+          if ((ell < xs->transfer_matrix.size()) && (apply_scatter_src) )
```

Comment:
Why is apply_scatter_src for?

Reply:
You probably mean "What is apply_scatter_src for?". It is a means to turn off certain portions of the `SetKSource` routine similar to what the `SetSource` routine does. The usefulness is not immediately apparent since we don't have any other methods other than `PowerIteration` but unless you have a strong objection against I would rather leave it untouched until we either can confirm it is not needed or find a better way to use the routine.